nix/build.nix

```diff
@@ -96,7 +96,7 @@ let
         ++ lib.optionals stdenv.hostPlatform.isDarwin [
           # TODO: move to overlay so it's usable in the shell
           (cargo-bundle.overrideAttrs (old: {
-            version = "0.6.0-zed";
```

Comment:
cc @P1n3appl3 I would merge the PR now to get Nightly back, but curious to see if this is the right change.

Reply:
https://github.com/zed-industries/zed/actions/runs/14061947731/job/39375455844 not well, so I'll revert this bit and will wait for professionals to update nix things then.